lib/sprockets/utils.rb

```diff
@@ -90,16 +90,18 @@ def string_end_with_semicolon?(str)
     end
 
     # Internal: Accumulate asset source to buffer and append a trailing
-    # semicolon if necessary.
+    # semicolon and newline. This may result in duplicated semicolons and
```

Comment:
We don't need to explain how it was before, so you can remove the section inside parentheses.